src/fiatExchanges/saga.ts

```diff
@@ -135,14 +135,14 @@ export function* tagTxsWithProviderInfo({ transactions }: UpdateTransactionsActi
     Logger.debug(`${TAG}@tagTxsWithProviderInfo`, `Checking ${transactions.length} txs`)
 
     const providerLogos: ProviderLogos = yield* select(providerLogosSelector)
-    const txHashesToProvider: TxHashToProvider = yield* call(fetchTxHashesToProviderMapping)
+    const txHashesToProvider: TxHashToProvider | null = yield* call(fetchTxHashesToProviderMapping)
```

Comment:
Can you instead fix the typing in `fetchTxHashesToProviderMapping`?

Reply:
yes good one, done